core/templates/dev/head/pages/exploration_editor/editor_tab/customize_interaction_modal_directive.html

```diff
@@ -27,14 +27,13 @@ <h3 ng-if="stateInteractionIdService.displayed && (customizationModalReopened ||
     </uib-tabset>
   </div>
 
-  <form ng-if="stateInteractionIdService.displayed && hasCustomizationArgs" name="form.schemaForm" class="protractor-test-interaction-editor">
+  <form ng-if="stateInteractionIdService.displayed && hasCustomizationArgs" name="form.schemaForm" class="interaction-editor-form protractor-test-interaction-editor">
     <div ng-repeat="customizationArgSpec in customizationArgSpecs track by $index">
-      <div class="oppia-interaction-customization-label"><[customizationArgSpec.description]></div>
-      <div>
-        <schema-based-editor local-value="stateCustomizationArgsService.displayed[customizationArgSpec.name].value" schema="customizationArgSpec.schema">
+      <div ng-if="customizationArgSpec.schema.type !== 'bool'" class="oppia-interaction-customization-label"><[customizationArgSpec.description]></div>
+        <schema-based-editor ng-class="{'boolean-checkbox': customizationArgSpec.schema.type === 'bool'}" local-value="stateCustomizationArgsService.displayed[customizationArgSpec.name].value" schema="customizationArgSpec.schema">
```

Comment:
Hi @srikar0896, sorry for missing this earlier, but shouldn't this line and the next one be de-indented by 2?

Reply:
@seanlip yes,thanks.Changed and submitted the changes.